frontend/src/dateUtil.ts

```diff
@@ -58,7 +58,7 @@ export function isToday(date: Date): boolean {
 }
 
 export function formatDate(date: Date): string {
-  return `${date.getDay()} ${date.toLocaleString('default', {
+  return `${date.getDate()} ${date.toLocaleString('default', {
```

Comment:
You sure you don't want to use "date.getTheRealFuckingDateNoIMeanForRealFuckOffJavascriptIHateYou()" instead?